app/models/article.rb

```diff
@@ -416,6 +416,19 @@ def access_by?(user)
     user.admin? || user_id == user.id
   end
 
+  def merge_with(merge_id)
+    begin
+      merge_art = Article.find(merge_id)
+      self.body += merge_art.body
+      self.comments += merge_art.comments
+      self.save
+      merge_art.delete
```

Comment:
The way this is written right now, you could potentially have a successful "merged" article saved in the database but then the delete might fail. Since both of these operations depend on one another, I would recommend looking into using a transaction, to ensure that either both actions happen, or both actions are rolled back.